src/components/TagList/TagList.js

```diff
@@ -96,7 +98,8 @@ export default class TagList extends Component {
             className="bx--tag-list--tag"
             type={tag.type}
             title={tag.name}
-            {...tag.otherProps}>
+            {...tag.otherProps}
+            {...(maxCharacters ? { maxCharacters } : {})}>
```

Comment:
Could this just be a prop instead of spread? e.g. `maxCharacters={maxCharacters}`

Reply:
Yeah, that makes sense. Initially, I wasn't sure if there was a use case for providing `maxCharacters` on a tag by tag basis. But on second thought I don't think there is. I'll go through and make this change.